apps/admin-gui/src/app/admin/pages/admin-page/admin-users/admin-users.component.html

```diff
@@ -18,6 +18,14 @@ <h1 class="page-subtitle">{{'ADMIN.USERS.TITLE' | translate}}</h1>
           (click)="onSearchByString()">
     <mat-icon>search</mat-icon>
   </button>
+  <button
+    mat-flat-button
+    color="primary"
+    class="mr-2"
```

Comment:
```suggestion
    mat-stroked-button
```
the red almost burned my eyes, otherwise its good

Reply:
red colour was removed to prevent further eye damage